src/backend/base/langflow/components/vectorstores/Redis.py

```diff
@@ -53,7 +55,7 @@ def build_vector_store(self) -> Redis:
                 documents.append(_input.to_lc_document())
             else:
                 documents.append(_input)
-        with open("docuemnts.txt", "w") as f:
+        with Path("docuemnts.txt").open("w") as f:
```

Comment:
typo in file name but I'm not sure of the impact if we change it...

Reply:
It seems like this is not being used anywhere